cyber-resilience-sig/whitepapers/stewards-and-cra.md

```diff
@@ -86,14 +86,40 @@ is followed. In the case the Project does not follow the Policy, the Steward nee
 As the Policy needs to be followed, the Steward and the Project will need to agree on a way to create the policy and provide changes.
 For example, accepting a generic Policy of a Foundation-Steward may be a requirement for creation of a project or accepting that project under that Foundation.
 
+#### A policy to develop secure products
+
+The legal text says *Open-source software stewards shall put in place and document in a verifiable manner a cybersecurity
+policy to foster the development of a secure product with digital elements* without further precisions. This section will need to be
+updated after the doubt of the meaning of this clause gets clarified, see [a dedicated FAQ entry](https://github.com/orcwg/cra-hub/issues/312).
+
+#### A policy to effectively handle vulnerabilities
+
+The policy to effectively handle vulnerabilities should be a part of the security policy, according to the Article 24(1), and include processes of:
+- documenting vulnerabilities - we understand this as documenting fixed or otherwise mitigated vulnerabilities, when they become public. Examples of techniques
+Projects may use are: CVE entries, security advisories (in text and/or machine-readable formats). All such vulnerabilities should be documented, and that
+documentation available from the same location that is easy to find for users. For example: it might be a link in `SECURITY.md` or on the Project's Security
+web page.
+- addressing vulnerabilities - the policy needs to define how the Project handles vulnerabilities, and it means the triaging and priritization. This is a part of a typical security process of open source projects.
+- remediating vulnerabilities - the policy needs to define how the Project remediates vulnerabilities, what means either fixes, or mitigations. This is a part of a typical security process of open source projects.
+- promoting information sharing in the open source community - this is a new item, which is currently usually implicit. Under the CRA, this part of the policy will need to
+be documented. The best practices include: Projects need to share information about found vulnerabilities
+with their upstream. Also, if a vulnerability report they receive has a cause in one of their dependencies, they need to notify the Project team of that dependency. And finally,
+is there is a probability of a similar issue in similar projects (for example in various HTTP implementations), Projects should notify the potentially affected peers they know about.
```

Comment:
" ... is there ... " -> "... if there ..."

Reply:
Fixed thanks